drivers/include/mpu9150.h

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    mpu9150 MPU-9150
```

Comment:
minor thing: could you change the group naming to `drivers_mpu9150` to conform to this discussion #2422?